CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# [Unreleased]
+# v0.3.0
 
 ## Added
```

Comment:
We should add a Changed section as well to specify:
- edition 2021
- the caret dependencies
- the display implementation of EventFd and Epoll now print the internal errors as well

Reply:
Done